### ISSUE TITLE
Kubify: Ability to choose LoadBalancer / ClusterIP / NodePort

### ISSUE THREAD

**Issue by cdrage** [containers/podman-desktop#1321]
### Is your enhancement related to a problem? Please describe

When deploying a service, we should have the ability to choose what kind of accessibility we want. 

It can either be:
* LoadBalancer (most common, managed by cloud or MetalLB)
* NodePort (second common, port above 3000)
* ClusterIP (internal IP, usually have to manual port forward for accessibility) 

### Describe the solution you'd like

UI the ability to choose between the three.
